### ISSUE TITLE
training problems

### ISSUE THREAD

**Issue by xz-123-new** [SherlockHolmes221/DOQ#7]
I got a problem with training this model, looking forward to your generous reply.      I train this model on hicodet dataset with resnet-50 as backbone ,   all the parameters concerning training are set following  the paper except batch size, i choose batch size =2 to train because  of GPU memory ,after i train 80 epoches and even 150 epoches now, i didn't get the mAP value it should be .For now, i only got ""test_mAP": 0.29916627302565096, "test_mAP rare": 0.23861787027422618, "test_mAP non-rare": 0.3172521595617908   ".Could you please  give me some advice about training stradegy to attain the correct number?thanks!

**Comment by SherlockHolmes221** [comment]
The batch size is set as 2 for per GPU by default. I do not understand your meaning.
 advice: the learning rate should be changed according to batch size
